cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/MockCruiseControl.java

```diff
@@ -74,9 +75,10 @@ public class MockCruiseControl {
             .endMetadata()
             .addToData("cruise-control.crt", MockCertManager.clusterCaCert())
             .build();
-    public static final Secret CC_API_SECRET = ModelUtils.createSecret(CruiseControlResources.apiSecretName(CLUSTER), NAMESPACE, Labels.EMPTY, null,
-            CruiseControl.generateCruiseControlApiCredentials(new PasswordGenerator(16), null, null), Collections.emptyMap(), Collections.emptyMap());
 
+    private static Map<String, String> apiSecretData = ApiCredentials.generateMapWithApiCredentials(ApiCredentials.generateCoManagedApiCredentials(new PasswordGenerator(16), null));
```

Comment:
Why not have this hardcoded? It does not seem like this thing is somehow dynamically changing